### PR TITLE
Setup data archiving for detached impressions

### PR DESCRIPTION
**Pull request by hopsoft**
# Feature

## Description

Add rake task that can be manually run to archive detached impressions tables.

## Why should this be added

This will ensure that we keep data size to a minimum and will help save hosting costs by limiting demand on data storage capacity.

### General

- [x] My code follows the style guidelines of this project & the tests are passing
- [x] I have performed a self-review of my own code
- [x] I have updated DB schema annotations or I have not made any DB schema changes

### CodeFund Core Team

- [x] I have added the appropriate milestone if applicable
- [x] I have requested a review from another developer(s)

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -2913,3 +2913,5 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20200221220825'),
 ('20200303224134'),
 ('20200325201726');
+
+
```
Comment (andrewmcodes) [inline db/structure.sql]: Nitpick but why is this added?

Comment (hopsoft) [reply db/structure.sql]: No idea. I generated a new local dev db and seeded it. Not sure why Rails decided to recreate this artifact with 2 new blank lines. 🤷‍♂
